app/api/services/users/create_user.py

```diff
@@ -0,0 +1,31 @@
+from uuid import uuid4
+
+from api.db.models.user_models import User, UserRole
+from api.route.api_context import ApiContext
+from api.route.models.user import UserParams, UserResponse
+
+
+def create_user(api_context: ApiContext) -> UserResponse:
```

Comment:
Todo either in this PR or a future one, but eventually the service layer shouldn't have any dependencies on the controller layer, so we'd want to slowly remove the dependency on ApiContext

Reply:
Very much agreed! Wasn't sure to what degree we wanted to add a bunch of TODOs to the code. I suppose I could handle this by creating git issues in the future!

Reply:
I've created a git issue, and added a TODO to these!